distribution/sdk/sample-mpegts-streamgenerator/src/main/java/org/codice/alliance/distribution/sdk/video/stream/mpegts/MpegTsUdpClient.java

```diff
@@ -94,28 +94,28 @@ public static void main(String[] args) {
         if (arguments.length == 1) {
             ip = DEFAULT_IP;
             port = DEFAULT_PORT;
-            LOGGER.warn("No IP or port provided.  Using defaults : {}:{}",
+            LOGGER.info("No IP or port provided.  Using defaults : {}:{}",
```

Comment:
Since the code already provides for fallback to defaults, these feel more like `debug` logs to me.